di-ipv-cimit-stub/lambdas/put-contra-indicator-credential/src/main/java/uk/gov/di/ipv/core/putcontraindicatorcredential/PutContraIndicatorsHandler.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.di.ipv.core.putcontraindicatorcredential;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.message.StringMapMessage;
+import uk.gov.di.ipv.core.putcontraindicatorcredential.domain.PutContraIndicatorsRequest;
+
+public class PutContraIndicatorsHandler
+        implements RequestHandler<PutContraIndicatorsRequest, String> {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    @Override
+    public String handleRequest(PutContraIndicatorsRequest event, Context context) {
+        LOGGER.info(new StringMapMessage().with("EVENT TYPE:", event.getClass().toString()));
+        return "Success";
```

Comment:
Return should be the json object 
```
{
  "result": "success"
}
```
Same with `PostMitigationsHandler`

Reply:
Done.

Reply:
Sorry mate. I meant a JSON object with a "result" key that (for this iteration) takes the value "success"